chaco/tests/test_colormapped_scatterplot.py

```diff
@@ -86,3 +86,11 @@ def test_colormap_updated(self):
         """ If colormapper updated then we need to redraw """
         self.color_mapper.updated = True
         self.assertFalse(self.scatterplot.draw_valid)
+
+    # regression test for enthought/chaco#425
+    def test_non_kiva_marker(self):
+        self.scatterplot.marker = "star"
+
+        self.gc.render_component(self.scatterplot)
+        actual = self.gc.bmp_array[:, :, :]
+        self.assertFalse(alltrue(actual == 255))
```

Comment:
I was being dumb before, it was trivial to write this regression test...

Reply:
Note this PR also fixes the test `test_scatter_custom` above once https://github.com/enthought/enable/pull/782 is pushed through that test can be unskipped.